mlflow/langchain/__init__.py

```diff
@@ -862,7 +862,7 @@ def autolog(
     log_model_signatures=False,
     log_models=False,
     log_datasets=False,
-    log_inputs_outputs=True,
+    log_inputs_outputs=False,
```

Comment:
Let's revert this, this default value has been discussed and we want to make it default to True to record user inputs & outputs, and it will serve as table for evaluation.

Reply:
If we revert this the default tracing experience creates a new run, which doesn't align with the decision.

> We don't create new runs automatically